testnet/install/mac.md

```diff
@@ -18,18 +18,22 @@ Tested on MacOS Sequoia Version 15.3.1 (Apple Silicon)
 
 Getting started with the Zond testnet Local system
 
-## 1. Install prerequisits 
+## 1. Install prerequisites 
 
-If you haven't done so already, install brew, then install [docker](https://docs.docker.com/desktop/setup/install/mac-install/) and other prerequesits with brew.
+If you haven't done so already, install brew, then install [docker](https://docs.docker.com/desktop/setup/install/mac-install/) and other prerequesites with brew.
 
 ```bash
 brew install bazel@7 kurtosis-tech/tap/kurtosis-cli jq yq
 ```
 
-Be sure to add bazel 7.5.0 to your path.
+Run this to make bazel 7 the default version (if other versions are installed)
+```bash
+brew unlink bazel@8 && brew link --force bazel@7
+```

```

Comment:
I had Bazel 8.1 already installed, and to override symlinks, this was required:
```bash
$ brew unlink bazel@8
Unlinking /opt/homebrew/Cellar/bazel/8.1.1... 1 symlinks removed.
$ brew link --force bazel@7
Linking /opt/homebrew/Cellar/bazel@7/7.5.0... 5 symlinks created.
```

Reply:
May have to add caveat about removing specific versions when guiding the user towards bazelisk